samples/templates/default-azuredeploy-docker.json

```diff
@@ -139,7 +139,7 @@
         },
         "sqlSchemaAutomaticUpdatesEnabled": {
             "type": "bool",
-            "defaultValue": "[if(equals(parameters('solutionType'),'FhirServerCosmosDB'), bool('false'), '')]",
+            "defaultValue": "[if(equals(parameters('solutionType'),'FhirServerCosmosDB'), bool('false'), bool('true'))]",
```

Comment:
@alsami  - Thanks for checking. We have added 'sqlSchemaAutomaticUpdatesEnabled' as an empty intentionally to let users decide consciously if they want to enable or disable automatic schema initialization based on their requirement. If we default to any(true\false), this may get overlooked and not get the desired behavior.

Reply:
@rbans96 I thought so but wasn't sure. Problem I see is that the error-messages are not very descriptive, something like 'expected type System.Bool but got System.String'.

Thanks for the heads up anyway!